crates/biome_js_formatter/src/context.rs

```diff
@@ -169,6 +169,9 @@ pub struct JsFormatOptions {
 
     /// Information related to the current file
     source_type: JsFileSource,
+
+    /// Enforce single attribute per line in HTML, Vue and JSX. Defaults to false
```

Comment:
I think it's better to remove `HTML` and `Vue` because the option is related only to JSX now.